config/sync/user.role.site_builder.yml

```diff
@@ -0,0 +1,117 @@
+uuid: 959f5492-24df-4212-9a9c-a0da88d464c4
+langcode: en
+status: true
+dependencies: {  }
+id: site_builder
+label: 'Site Builder'
+weight: -1
+is_admin: null
+permissions:
+  - 'access administration pages'
+  - 'access config_pages overview'
+  - 'access content overview'
+  - 'access contextual links'
+  - 'access devel information'
+  - 'access entity usage statistics'
+  - 'access environment indicator'
+  - 'access file_browser entity browser pages'
+  - 'access files overview'
+  - 'access image_browser entity browser pages'
+  - 'access media overview'
+  - 'access media_browser entity browser pages'
+  - 'access patterns page'
+  - 'access site in maintenance mode'
+  - 'access site reports'
+  - 'access taxonomy overview'
+  - 'access user profiles'
+  - 'access video_browser entity browser pages'
+  - 'admin classes'
+  - 'admin display suite'
+  - 'admin fields'
+  - 'administer block_content display'
+  - 'administer block_content fields'
+  - 'administer block_content form display'
+  - 'administer blocks'
+  - 'administer content types'
+  - 'administer crop'
+  - 'administer crop types'
+  - 'administer display modes'
+  - 'administer entity browsers'
+  - 'administer footer menu items'
+  - 'administer image styles'
+  - 'administer main menu items'
+  - 'administer menu'
+  - 'administer modules'
+  - 'administer node display'
+  - 'administer node fields'
+  - 'administer node form display'
+  - 'administer nodes'
+  - 'administer paragraph display'
+  - 'administer paragraph fields'
+  - 'administer paragraph form display'
+  - 'administer paragraphs settings'
+  - 'administer paragraphs types'
+  - 'administer redirects'
+  - 'administer responsive images'
+  - 'administer shortcuts'
+  - 'administer taxonomy'
+  - 'administer taxonomy_term display'
+  - 'administer taxonomy_term fields'
+  - 'administer taxonomy_term form display'
+  - 'administer themes'
+  - 'administer url aliases'
+  - 'administer user display'
+  - 'administer user fields'
+  - 'administer user form display'
+  - 'administer users'
```

Comment:
I think we need to turn on and enable role delegation too. This permission allows them to add a sunet user with any role

Reply:
🤔 This sounds like privilege escalation and should be stopped. Thanks. I'll revise.